src/wizard/android/index.md

```diff
@@ -69,6 +79,40 @@ class MyActivity : AppCompatActivity() {
 }
 ```
 
+### Performance Monitoring
+
+Set `io.sentry.traces.sample-rate` to 1.0 to capture 100% of transactions for performance monitoring.
+
+We recommend adjusting this value in production.
+
+```xml {filename:AndroidManifest.xml}
+<application>
+    <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+</application>
+```
+
+You can measure the performance of your code by capturing transactions and spans.
+
+```kotlin
+import io.sentry.Sentry
+
+// Transaction can be started by providing, at minimum, the name and the operation
+val transaction = Sentry.startTransaction("updateRepos", "task")
+
+// Transactions can have child spans (and those spans can have child spans as well)
+val span = transaction.startChild("test-child-operation")
```

Comment:
l: Maybe we should choose better operation names, as they are used for filtering.
```suggestion
val transaction = Sentry.startTransaction("Update Repos", "db")

// Transactions can have child spans (and those spans can have child spans as well)
val span = transaction.startChild("db", "Update first repo")
```